1030-stepper-design-guidance.md

```diff
@@ -0,0 +1,31 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-16
+updated: 2024-10-21
+---
+
+# Stepper Component
+
+The [Stepper](https://clarity.design/documentation/stepper) is an extension of the base accordion component. It structures multi-step processes into two or more expanding panels that break up complex workflows.
```

Comment:
```suggestion
The [stepper](https://clarity.design/documentation/stepper) is an accordion-like component that structures multi-step processes into two or more expanding panels that break up complex workflows.
```

The fact that the step is an "extension" of the accordion is an implementation detail that will hopefully not be true in the future.